.gitignore

```diff
@@ -222,3 +222,4 @@ Event-Site/packages/RavenDB.Server.2.0.2330/tools/Database/
 
 #ImageStorage
 /EventSite/Content/localImageStorage/
+/.vs/config
```

Comment:
we shiould just ignore at the root .vs level, which I added earlier, but this is OK for now.